src/ocean/meta/types/Qualifiers.d

```diff
@@ -61,12 +61,16 @@ alias char[]         mstring;
 
 template Const(T)
 {
-    alias const(T) Const;
+    // workaround for deprecating Const
+    // https://issues.dlang.org/show_bug.cgi?id=10181
+    deprecated("Use const")
+    alias const(T) Temp;
+    alias Temp Const;
 }
 
 unittest
 {
-    alias Const!(int[]) Int;
+    alias const(int[]) Int;
```

Comment:
That unittest is kinda useless now. And I suppose you can't mark the unittest as `deprecated` ?

Reply:
I can also remove the unittest.